Jenkinsfile

```diff
@@ -101,31 +101,19 @@ pipeline {
             params.EE_BRANCH_NAME = 'cambpm-ee-main/pipeline-master'
           } else {
             labels = JsonOutput.toJson(pullRequest.labels)
+            shaCE = sh(script:"git rev-parse HEAD", returnStdout: true).trim()
           }
 
           if (cambpmWithLabels('webapp-integration','all-as','h2','websphere','weblogic','jbosseap','run','spring-boot','authorizations')) {
```

Comment:
We should add a `master` label here. Otherwise, the EE branch won't be triggered on the `master` branch.

Reply:
As we discussed it will be triggered even without the master label.